src/build.jl

```diff
@@ -0,0 +1,154 @@
+# This file generated the Markdown documentation files.
+
+# The @file macro generates the documentation for a particular file
+# where the {{method1, methods2}} includes the documentation for each method
+# via the `buildwriter` function.
+
+# Currently this prints the methodtable followed by the docstring.
+
+macro file(fn, md_string, m::Module) buildfile(fn, md_string, m) end
+
+function generate_for(m::Module, file_names...; doc_dir="docs", gen_dir="_generated")
+    eval(Expr(:toplevel, quote using $(symbol(m)) end))
```

Comment:
In 0.3 ``using`` can't be put directly into quotes. I believe it was a fixed quite recently. You'd need to used ``Expr(:using, ...)`` instead until we've moved to 0.4/0.5.